src/Tribe/API.php

```diff
@@ -480,11 +480,12 @@ protected static function prepare_event_date_meta( $event_id, $data ) {
 		 * leveraged.
 		 *
 		 * @param $event_id
+		 * @param array $default The default cost for the Event.
 		 */
-		public static function update_event_cost( $event_id ) {
+		public static function update_event_cost( $event_id, $default = [] ) {
 			// Loads current event costs, on construct
 			// Tribe__Events__Tickets__Tickets->get_ticket_prices() adds them to this filter
-			$event_cost = (array) apply_filters( 'tribe_events_event_costs', [], $event_id );
+			$event_cost = (array) apply_filters( 'tribe_events_event_costs', $default, $event_id );
```

Comment:
Needs a docblock

Reply:
Yeah, sorry - I operate under the "if you change it you own it" 😉 rule - it was not that you did something wrong @EugeneKyale